src/test/java/org/jabref/logic/importer/fileformat/PdfMergeMetadataImporterTest.java

```diff
@@ -95,4 +95,20 @@ void importWorksAsExpected() throws Exception {
 
         assertEquals(Collections.singletonList(expected), result);
     }
+
+    @Test
+    void pathShouldBeRelativized() throws Exception {
+        Path file = Path.of(PdfMergeMetadataImporter.class.getResource("/pdfs/minimal.pdf").toURI());
+        Path workingDir = Path.of(PdfMergeMetadataImporter.class.getResource("/pdfs/").toURI());
+        List<BibEntry> result = importer.importDatabase(file, workingDir).getDatabase().getEntries();
+
+        BibEntry expected = new BibEntry(StandardEntryType.InProceedings);
+        expected.setField(StandardField.AUTHOR, "1 ");
+        expected.setField(StandardField.TITLE, "Hello World");
```

Comment:
Try to use the JabRef style:

```suggestion
        BibEntry expected = new BibEntry(StandardEntryType.InProceedings)
        .withField(StandardField.AUTHOR, "1 ")
        .withField(StandardField.TITLE, "Hello World");
```

(also rewrite next line with the `.with...` pattern